admin/qtx_admin_settings.php

```diff
@@ -259,6 +259,7 @@ private function add_sections( $nonce_action ) {
         $admin_sections['import']      = __( 'Import', 'qtranslate' ) . '/' . __( 'Export', 'qtranslate' );
         $admin_sections['languages']   = __( 'Languages', 'qtranslate' );
 
+        //TODO: this actually assumes every manual activation module has settings, dedicated key to be added if that is not the case...
```

Comment:
The activation setting has to be saved differently. I may revisit the `ma_module_enabled` config.